modules/nf-core/bowtie2/build/main.nf

```diff
@@ -1,6 +1,6 @@
 process BOWTIE2_BUILD {
     tag "$fasta"
-    label 'process_high'
```

Comment:
The files in `modules/nf-core` are automatically fetched from https://github.com/nf-core/modules/ and should not be modified manually.

Instead, you should specify the new label in `base.config`. See this example from Sarek: https://github.com/nf-core/sarek/blob/master/conf/base.config#L74-L77